.gitignore

```diff
@@ -15,8 +15,6 @@ dist/
 downloads/
 eggs/
 .eggs/
-lib/
-lib64/
```

Comment:
caused the new js folder to be ignored inadvertently and our python deps are stored outside this repository so no need to ignore